src/routes/auth/Auth.js

```diff
@@ -0,0 +1,75 @@
+import React, { useState, useCallback } from 'react';
+import withStyles from 'isomorphic-style-loader/withStyles';
+import s from './Auth.css';
+
+function Auth() {
+  const [inputValue, setInputValue] = useState({
```

Comment:
Inconvenient, right? I could suggest using `useReducer` here, but I don't insist as we plan to switch to Redux soon.